ModernFlyouts/Properties/Strings.resx

```diff
@@ -130,7 +130,7 @@
     <value>Dependencies</value>
   </data>
   <data name="About.FileABug" xml:space="preserve">
-    <value>If you find any bugs, please open a new issue in the github repository.</value>
+    <value>Jeśli znajdziesz jakieś błędy, otwórz nowe issue w repozytorium aplikacji w serwisie GitHub.</value>
```

Comment:
This shouldnt be in english? since its in the original Strings(I messed up with the previous review and i didnt want to do it,sry)